### PR TITLE
Update to gmt-6.0.0a19 on Linux and macOS

### PR DESCRIPTION
**Pull request by seisman**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by seisman** [comment]
@conda-forge-admin, please rerender

**Comment by seisman** [comment]
This PR is trying to rebuild the GMT master branch again.

This branch is created on top of the devel branch, with the changes in the master branch merged in. Although there are a lot of commits in this RR, the differences between this branch and the master branch are very tiny. See the output of `git diff master gmt-6.0.0a19` for the differences.

The linux and osx builds succeed. However, the Windows build fails, which tries to read `gmt/share` files in the **build environment** instead of the **test environment**.

**Comment by leouieda** [comment]
That's very strange. The "failure" seems to be relate to the custom PS fonts (which was recently introduced to 6). But this failure happens on Linux and Mac as well. Also, there is no error message so this looks like a crash to me.

**Comment by seisman** [comment]
@conda-forge-admin, please rerender

**Comment by seisman** [comment]
@leouieda Currently, I don't have any clues to solve the Windows build failure. What about skipping the Windows build of GMT 6.0.0?

**Comment by leouieda** [comment]
@seisman this would be good to have for testing. I'm OK with removing Windows for now. We can open another PR to try to diagnose this before releasing GMT6. It's good that we caught this now and not after the release.

**Comment by leouieda** [comment]
If you want, please feel free to add yourself as a recipe maintainer in `meta.yaml` so that you can have merge rights on this repo. Keep in mind that we always have to use forks in conda-forge, though. All branches get built and uploaded so even maintainers work from forks to avoid this.

**Comment by seisman** [comment]
I've added myself as a maintainer. Before merging, can you check if the build will be uploaded to the dev channel?

File: conda-forge.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
+  secure: {BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1}
+compiler_stack: comp7
+max_py_ver: '37'
+max_r_ver: '35'
+travis:
+  secure: {BINSTAR_TOKEN: AVAEOvgJRTbM1j5fpK4p4DxAZlCxMjFbuWgi4p3Sz1XpV51yL7rFD0xYo6Bie+JAfRZeja4mJ6rwm+LaAYA6Xvl536jYVFLS4WSw0yIdFgo+cOHp3ZPIEMhLN7Aag4IeyRyPoIAFcDAMNGyP/kJCK07+dUBSDRuHYqC0LjToz1HlaPmWC1VHWqHAu2ENGyoxa37Mzaehut/kQrh1qHBOzV4YZYipw9W9oRauLmulL7HYlUJKtfchb58ifakIo4EdhDX8c5oI9EetV+skoOSso9Gg9wQRb3XWQxup4D2M/dMJ7gYRpE6Umc7LMKK24ilSFS0nXjIj9jOkEjmUP5jDbf6dih8wlwgwsMNf9BP38hZ/bTak32TUKMEMwc23DbOjzkzWbanwtJ8eafY1GVdwWzctygYT2NoGoDAZKUQ5PHVa+Keyy6LQiz5boIY243q8odrG2o20gDvyrBxRT5/LBbE98IB23wr3VoCiV53Mir3swRCfRhV8VxqYp3UeC8szYjF+EsLAF8Gh4DX9OWsLoihppjR3Vcrl4LcH1Am6bBH4m1xJs8rOINP0ILBJTlJ00qH//WA/9451ZBeYcvWA50kWJ3ApjkmBhyny8XaJKanB12FEtcswcnepAld2ZCsKhPjZtRgDmh5dujB/wvKV31F5lbemsA0lZDFXto8oZpo=}
 channels:
   targets:
-    - [conda-forge, dev]
+    - ["conda-forge", "dev"]
```
Comment (leouieda) [inline conda-forge.yml]: It seems like the upload will be to the dev label on conda-forge. See http://conda-forge.org/docs/maintainer/conda_forge_yml.html#channels It should be fine.

**Comment by leouieda** [comment]
@conda-forge-admin, please rerender

**Comment by seisman** [comment]
@leouieda Something goes wrong. The gmt-6.0.0a19 was pushed to the main channel. See https://anaconda.org/conda-forge/gmt/files

**Comment by leouieda** [comment]
I noticed. That sucks. I wonder if we forgot to change something. @ocefpaf did anything change with regard to using different labels on anaconda.org?

**Comment by ocefpaf** [comment]
We changed the tools and things things are still in flux at the moment. See https://github.com/conda-forge/conda-smithy/issues/897

I can relabel that manually for now.

**Comment by seisman** [comment]
@ocefpaf Thanks for help relabel the releases.

Are there any workarounds? Does it work if I manually edit the `channel_targets` in `.ci_support/linux_.yaml` and `.ci_support/osx_.yaml` after conda-forge-admin's rerendering?

**Comment by leouieda** [comment]
Thanks, Filipe! I'll keep an eye on that issue as well

**Comment by ocefpaf** [comment]
> @ocefpaf Thanks for help relabel the releases.

No problem.
 
> Are there any workarounds?

I believe that setting the `conda_build_config.yaml` would fix it.
(We actually need to test and document that properly.)

**Comment by leouieda** [comment]
Thanks for the tips. Where is that file? From the docs, it seems like it is taken from the conda install. Can we set it in the repo?

**Comment by ocefpaf** [comment]
There is a global one but you can augment it with another one in the recipe directory. I believe you do no need to copy everything from the global file, just add the extras parts, in this case the channel/label configuration.

I think that the right way to do this is:

```yaml
channel_targets:
  - conda-forge main
  - conda-forge dev
```

but I never really tested it.

**Pull request merged**